### PR TITLE
Add togglepassphrase function

### PR DESCRIPTION
**Pull request by wszdexdrf**

### CODE REVIEW

**Comment by wszdexdrf** [comment]
Should this test be ignored? [`toggle_passphrase`](https://hwi.readthedocs.io/en/stable/devices/index.html#hwilib.devices.ledger.LedgerClient.toggle_passphrase) is not supported by Ledger.

File: src/lib.rs [hunk 1]
```diff
@@ -227,4 +227,19 @@ mod tests {
         HWIClient::set_log_level(types::LogLevel::DEBUG).unwrap();
         test_enumerate();
     }
+
+    #[test]
+    #[serial]
+    fn test_toggle_passphrase() {
+        let devices = HWIClient::enumerate().unwrap();
+        for device in devices {
+            if device.device_type == "ledger" {
```
Comment (danielabrozzoni) [inline src/lib.rs:236]: ACK for the approach, but the ledger is not the only device with this problem. It's the only device **we test** with this problem.

https://hwi.readthedocs.io/en/latest/devices/index.html#support-matrix

Comment (wszdexdrf) [reply src/lib.rs:236]: That is correct. I was hoping when we add new devices, we would modify the conditions at the same time.

**Comment by danielabrozzoni** [comment]
ACK fe184f11084756e61315b7afe3224d12561d0797

**Pull request merged**